### PR TITLE
Einbindung KWL recoVAIR 260/4

### PR DESCRIPTION
**Pull request by Skibbie**

### CODE REVIEW

File: ebusd-2.1.x/de/vaillant/08.recov.csv [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+*r,,,,,,B509,,,,IGN:2,,,
+*w,,,,,,B509,,,,,,,
+r,recov,BypassPosition,Position der Bypass-Klappe,,,,290b0e,,,UIN,10,,
+r,recov,TempOutsideAir,Temperatur Aussenluft,,,,290c0e,,,tempsensor,,,
+r,recov,TempOutgoingAir,Temperatur Fortluft,,,,290d0e,,,tempsensor,,,
+r,recov,TempWasteAir,Temperatur Abluft,,,,29b40d,,,tempsensor,,,
+r,recov,TempInletAir,Temperatur Zuluft,,,,29b30d,,,tempsensor,,,
+r,recov,HumiWasteAir,Luftfeuchtigkeit Zuluft,,,,298503,,,percent,,,
+r,recov,FrostProtection,Frostschutzelement aktiv,,,,29f10c,,,onoff,,,
+r;w,recov,PreferredTemp,Wunschtemperatur,,,,29140e,,,temp,,,
```
Comment (john30) [inline ebusd-2.1.x/de/vaillant/08.recov.csv]: is that really writable? please verify

File: ebusd-2.1.x/de/vaillant/08.recov.csv [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,recov,recoVair 260,,,,,,,,,,,
+# ##### Generell #####,,,,,,,,,,,,,
+*r,,,,,,B513,,,,IGN:2,,,
+*w,,,,,,B509,,,,,,,
+r,recov,VentIntensityDay,Lueftungsintensitaet Tag auslesen,,,,048a03,,,UCH,,,
+r,recov,VentIntensityNight,Lueftungsintensitaet Nacht auslesen,,,,048b03,,,UCH,,,
+r,recov,VentHeatRecovery,Waermerueckgewinnung auslesen,,,,048c03,,,UCH,0=auto;1=an;2=aus,,
+w,recov,SetVentIntensityDay1,Lueftungsintensitaet Tag 1 setzen,,,,0e8a0301,,,,,,
```
Comment (john30) [inline ebusd-2.1.x/de/vaillant/08.recov.csv]: this should be replaced with a intensity parameter and combined with the read

File: ebusd-2.1.x/de/vaillant/08.recov.csv [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+w,recov,SetVentIntensityDay2,Lueftungsintensitaet Tag 2 setzen,,,,0e8a0302,,,,,,
+w,recov,SetVentIntensityDay3,Lueftungsintensitaet Tag 3 setzen,,,,0e8a0303,,,,,,
+w,recov,SetVentIntensityDay4,Lueftungsintensitaet Tag 4 setzen,,,,0e8a0304,,,,,,
+w,recov,SetVentIntensityDay5,Lueftungsintensitaet Tag 5 setzen,,,,0e8a0305,,,,,,
+w,recov,SetVentIntensityDay6,Lueftungsintensitaet Tag 6 setzen,,,,0e8a0306,,,,,,
+w,recov,SetVentIntensityNight1,Lueftungsintensitaet Nacht 1 setzen,,,,0e8b0301,,,,,,
```
Comment (john30) [inline ebusd-2.1.x/de/vaillant/08.recov.csv]: same as above

File: ebusd-2.1.x/de/vaillant/08.recov.csv [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+w,recov,SetVentIntensityNight2,Lueftungsintensitaet Nacht 2 setzen,,,,0e8b0302,,,,,,
+w,recov,SetVentIntensityNight3,Lueftungsintensitaet Nacht 3 setzen,,,,0e8b0303,,,,,,
+w,recov,SetVentIntensityNight4,Lueftungsintensitaet Nacht 4 setzen,,,,0e8b0304,,,,,,
+w,recov,SetVentIntensityNight5,Lueftungsintensitaet Nacht 5 setzen,,,,0e8b0305,,,,,,
+w,recov,SetVentIntensityNight6,Lueftungsintensitaet Nacht 6 setzen,,,,0e8b0306,,,,,,
+w,recov,SetVentHeatRecoveryAuto,Waermerueckgewinnung Auto,,,,0e8c0300,,,,,,
```
Comment (john30) [inline ebusd-2.1.x/de/vaillant/08.recov.csv]: same as above

File: ebusd-2.1.x/de/vaillant/08.recov.csv [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,recov,recoVair 260,,,,,,,,,,,
+# ##### Generell #####,,,,,,,,,,,,,
+*r,,,,,,B513,,,,IGN:2,,,
+*w,,,,,,B509,,,,,,,
+r,recov,VentIntensityDay,Lueftungsintensitaet Tag auslesen,,,,048a03,,,UCH,,,
```
Comment (john30) [inline ebusd-2.1.x/de/vaillant/08.recov.csv]: the circuit can be left out when either the file is intended fro use with scanconfig or the default lines contain the circuit

**Comment by kmplngj** [comment]
What is missing to merge this? I also have a recoVAIR 260 and like to use the config. Is there something I could help to fix this?

**Comment by Eraser-1** [comment]
> What is missing to merge this? I also have a recoVAIR 260 and like to use the config. Is there something I could help to fix this?

The same for me

**Comment by Eraser-1** [comment]
Why isn't the csv-file at "ebusd-2.1.x/de/vaillant/08.recov.csv"?
From where to download? Or were there some problems with it?

**Comment by kmplngj** [comment]
> Why isn't the csv-file at "ebusd-2.1.x/de/vaillant/08.recov.csv"?
> From where to download? Or were there some problems with it?

It is not merged yet :(

You could find it in this pull request. Link to file: https://raw.githubusercontent.com/john30/ebusd-configuration/429e6e95b541b6cf13b29ab27b96547f82a4afdc/ebusd-2.1.x/de/vaillant/08.recov.csv

**Comment by straccio** [comment]
If you have 15.700 you can use it to control the VMC with this https://github.com/john30/ebusd-configuration/pull/191

**Comment by Eraser-1** [comment]
Thanks to both of you!
If I understand correct, on every way I choose with either using the 08.recov.csv or changing the 15.700.csv, I have to change the config-file-path of my ebusd-installation to local and so then I will have no direct online-configuration-updates anymore, correct?

**Comment by john30** [comment]
whats missing is that someone is implementing the requested changes from above and also supplies an english version

**Comment by kmplngj** [comment]
@john30 I like to fix the open issues. I have ebusd running and added the file to my configuration folder. After a scan the config file could not be loaded because it does not seem to match. 

ebusd    | 2021-07-19 21:56:33.858 [bus notice] new master 03, master count 2
ebusd    | 2021-07-19 21:56:33.858 [update info] sent MS cmd: 3108070400 / 0ab57265636f5602250203
ebusd    | 2021-07-19 21:56:33.858 [bus notice] scan 08: ;Vaillant;recoV;0225;0203
ebusd    | 2021-07-19 21:56:33.858 [update notice] store 08 ident: done
ebusd    | 2021-07-19 21:56:33.858 [update notice] sent scan-read scan.08  QQ=31: Vaillant;recoV;0225;0203
ebusd    | 2021-07-19 21:56:33.858 [bus debug] notify request: done
ebusd    | 2021-07-19 21:56:33.858 [bus notice] scan 08: ;Vaillant;recoV;0225;0203
ebusd    | 2021-07-19 21:56:33.858 [bus debug] switching from send response ACK to send SYN
ebusd    | 2021-07-19 21:56:33.858 [main debug] <<< 08;Vaillant;recoV;0225;0203
ebusd    | 2021-07-19 21:56:33.868 [bus debug] send/receive symbol latency 9 ms
ebusd    | 2021-07-19 21:56:38.859 [main debug] performing regular tasks
ebusd    | 2021-07-19 21:56:38.860 [main debug] found 3 matching scan config files from vaillant with prefix 08: done
ebusd    | 2021-07-19 21:56:38.860 [main error] unable to load scan config 08: no file from vaillant with prefix 08 matches ID "recov", SW0225, HW0203
ebusd    | 2021-07-19 21:56:38.860 [main error] scan config 08: ERR: element not found
ebusd    | 2021-07-19 21:56:38.860 [mqtt debug] publish ebusd/global/scan running

File: ebusd-2.1.x/de/vaillant/08.recov.csv (hunk 1 above)
Comment (kmplngj) [reply ebusd-2.1.x/de/vaillant/08.recov.csv:37]: No, it is readonly.
```suggestion
r,recov,PreferredTemp,Wunschtemperatur,,,,29140e,,,temp,,,
```

File: ebusd-2.1.x/de/vaillant/08.recov.csv (hunk 2 above)
Comment (kmplngj) [reply ebusd-2.1.x/de/vaillant/08.recov.csv:9]: How could I to this?

File: ebusd-2.1.x/de/vaillant/08.recov.csv (hunk 3 above)
Comment (kmplngj) [reply ebusd-2.1.x/de/vaillant/08.recov.csv:15]: same as above. How could I to this?

File: ebusd-2.1.x/de/vaillant/08.recov.csv (hunk 5 above)
Comment (kmplngj) [reply ebusd-2.1.x/de/vaillant/08.recov.csv:6]: @john30  I don't understand the comment. I'm too dumb for that...

**Comment by john30** [comment]
abandoned in favor of #343

**Pull request closed without merging**